packages/ui/src/common/email-or-phone-number-field.tsx

```diff
@@ -13,20 +13,16 @@ export function EmailOrPhoneNumberField({
   labelEmail = 'Email address',
   labelPhoneNumber = 'Phone number',
   locationBasedCountryIso,
-  requiredEmail,
-  requiredPhoneNumber,
```

Comment:
Removing the separate required props as there is not a use where one is required and the other is not in sign in flow.